Merged with updated main branch w/ all previous PR changes. Merge is broken:

Map.js:128 Error during search: AuthenticationError: 401 Incorrect API key provided: sk-proj-********************************************************************************************************************************************************4gEA. You can find your API key at https://platform.openai.com/account/api-keys.
    at APIError.generate (error.ts:74:1)
    at OpenAI.makeStatusError (core.ts:431:1)
    at OpenAI.makeRequest (core.ts:495:1)
    at async getSynonyms (GPTRequest.js:6:1)
    at async handleSubmit (Map.js:67:1)

--- a/locationsearch/client/src/components/GPTRequest.js
+++ b/locationsearch/client/src/components/GPTRequest.js
@@ -1,3 +1,4 @@
+/*
 import OpenAI from "openai";
 
 const openai = new OpenAI({ apiKey: "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", dangerouslyAllowBrowser: true });
@@ -39,3 +40,4 @@ export async function getSynonyms(initWord) {
 
 //     return cleanItems;
 // }
+*/

--- a/locationsearch/client/src/components/Map.js
+++ b/locationsearch/client/src/components/Map.js
@@ -71,6 +71,10 @@ function Map() {
         searchTypes.push(adjustedType.trim().toLowerCase().replace(/\s+/g, '_'));
       }
       */
+
+      if (placeType.trim() !== "") {
+        searchTypes.push(placeType.trim().toLowerCase().replace(/\s+/g, '_'));
+      }
      
       let foundPlaces = await nearbySearch(lat, lng, rad, searchTypes);
 

--- a/locationsearch/client/src/components/Map.js
+++ b/locationsearch/client/src/components/Map.js
@@ -1,6 +1,6 @@
 import React, { useState, useEffect, useRef } from "react";
 import { initMap, centerMap, processPlaces, nearbySearch, detailSearch, MI_TO_METERS, haversine_distance } from "../components/MapUtils";
-import { getSynonyms } from "../components/GPTRequest";
+//import { getSynonyms } from "../components/GPTRequest";
 
 function Map() {
   const [latitude, setLatitude] = useState(33.7756);
@@ -64,12 +64,14 @@ function Map() {
       
       const searchTypes = [...placeTypes];
 
+      /*
       let adjustedType = await getSynonyms(placeType);
 
       if (adjustedType.trim() !== "") {
         searchTypes.push(adjustedType.trim().toLowerCase().replace(/\s+/g, '_'));
       }
-      
+      */
+     
       let foundPlaces = await nearbySearch(lat, lng, rad, searchTypes);
 
       foundPlaces.forEach((place) => {
@@ -278,8 +280,8 @@ function Map() {
                         'N/A'
                       )}
                     </td>
-                    <td className="border px-6 py-4 text-lg">{place.price_level}</td>
                     <td className="border px-6 py-4 text-lg">{place.distance.toFixed(2)}</td>
+                    <td className="border px-6 py-4 text-lg">{place.price_level}</td>
                   </tr>
                 ))}
               </tbody>

